gamearea.py

```diff
@@ -268,7 +268,7 @@ def __release_cb(self, widget, event):
         self.selected_cat = None
         if "type" not in self.level_data.keys():
             return 0
-        if self.level_data["type"] == GameType.ROWS:
+        if self.level_data["type"] == GameType.ROWS or self.level_data["type"] == GameType.CHOOSE:
```

Comment:
The commit message says " Add functionlity of _release_cb in gameType CHOOSE" but the change in the commit doesn't reflect that.

Reply:
Yes, I realised it was quite confusing.

I have changed the commit message [7f629b2](https://github.com/sugarlabs/classify-cats/pull/9/commits/7f629b24a274f77578c2e64c7cef418a725eb857)